pkg/dom0-ztools/rootfs/bin/mdev-zvol.sh

```diff
@@ -0,0 +1,19 @@
+#!/bin/sh
+
+if [ "$ACTION" = "add" ]; then
+  datasetName=$(/lib/udev/zvol_id "/dev/$MDEV")
+  retVal=$?
+  if [ $retVal -ne 0 ]; then
+    echo "$datasetName"
+    exit $retVal
+  fi
+  #temp directory to map $MDEV->datasetName
+  mkdir -p "/run/mdev/zvol"
+  echo "$datasetName" >"/run/mdev/zvol/$MDEV"
+  mkdir -p "$(dirname "/dev/zvol/$datasetName")"
+  ln -s "/dev/$MDEV" "/dev/zvol/$datasetName"
+else
```

Comment:
Wouldn't it make sense to have an explicit action for "delete"?

Reply:
+1 to @eriknordmark

Reply:
Good point, reworked this.